README.md

```diff
@@ -15,7 +15,7 @@ grant imported privileges on database snowflake to role your_dbt_role_name;
 ```yaml
 packages:
   - package: get-select/dbt_snowflake_monitoring
-    version: 4.0.2
+    version: 4.1.0
```

Comment:
not a breaking change since 

```yml
models:
  +on_schema_change: "append_new_columns"
```

https://docs.getdbt.com/docs/build/incremental-models#what-if-the-columns-of-my-incremental-model-change